hydra-node/test/Hydra/Ledger/Builder.hs

```diff
@@ -0,0 +1,17 @@
+-- | Provides utility functions to build transaction and other data types
+-- useful for testing purpose
+module Hydra.Ledger.Builder where
```

Comment:
This is only about building `SimpleTx` right? If yes, I'd say the module name should include that somehow

Reply:
Also, these are only used in tests and there might be an overlap with the `Gen` functions. So maybe this warrants a `Test.Ledger.Simple`?

Reply:
Ahh.. it is a module in the tests already. Yeah, so was not so obvious to me :sweat_smile: I guess I am used to read a `Test.` in a test-only module

Reply:
About the the fact it's dedicated to `SimpleTx`: Right now, yes it is, but the idea of this builder construction is to decouple the tests from the way to build the data so that in the tests you can use more "abstract" functions without needing to know the details of how they are constructed.
How about renaming the module`Hydra.Ledger.TestTxBuilder`?